src/components/AddEntryForm.js

```diff
@@ -30,21 +31,12 @@ class AddEntryForm extends Component {
             true,
         )
     }
-    getCurrentDate = () => {
-        let today = new Date()
-        let dd = today.getDate()
-        let mm = today.getMonth() + 1 //January is 0!
 
-        let yyyy = today.getFullYear()
-        if (dd < 10) {
-            dd = "0" + dd
-        }
-        if (mm < 10) {
-            mm = "0" + mm
-        }
-        let date = dd + "/" + mm + "/" + yyyy
-        return date
+    getCurrentDate = () => {
+        // "02/01/2018"
+        return moment(new Date()).format('DD/MM/YYYY')
```

Comment:
Oh, this is nice! I've never used Moment before.

Reply:
Moment is super handy. Once you have used it, you never have to deal with dates on your own again.